handlers.go

```diff
@@ -0,0 +1,27 @@
+package telebot
+
+// Default handler prefix.
+const Default string = ""
```

Comment:
This constant is never used

Reply:
I added it for convenience, so I could write:
```go
b.Handle(tb.Default, func(c Context) {})
```

Reply:
Ah makes sense.